src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt

```diff
@@ -0,0 +1,152 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+internal class SentenceCalculationTest {
+
+  @Test
+  fun calculatedReleaseDate() {
+    val duration = Duration(2.0, ChronoUnit.DAYS)
+    val sentencedAt: LocalDate = LocalDate.of(2020, 1, 1)
+    val sentence = Sentence(duration, sentencedAt, 0, 0)
+    val sentenceCalculation = SentenceCalculation(sentence)
+    val calculatedReleaseDate: LocalDate = LocalDate.of(2020, 1, 1)
+    sentenceCalculation.calculatedReleaseDate = calculatedReleaseDate
+    assertEquals(calculatedReleaseDate, sentenceCalculation.calculatedReleaseDate)
+  }
+
+  @Test
+  fun calculatedExpiryDate() {
+    val duration = Duration(2.0, ChronoUnit.DAYS)
+    val sentencedAt: LocalDate = LocalDate.of(2020, 1, 1)
+    val sentence = Sentence(duration, sentencedAt, 0, 0)
+    val sentenceCalculation = SentenceCalculation(sentence)
+    val calculatedExpiryDate: LocalDate = LocalDate.of(2020, 1, 1)
+    sentenceCalculation.calculatedExpiryDate = calculatedExpiryDate
+    assertEquals(calculatedExpiryDate, sentenceCalculation.calculatedExpiryDate)
+  }
+
+  @Test
+  fun numberOfDaysToSentenceExpiryDate() {
+    val duration = Duration(2.0, ChronoUnit.DAYS)
+    val sentencedAt: LocalDate = LocalDate.of(2020, 1, 1)
+    val sentence = Sentence(duration, sentencedAt, 0, 0)
+    val sentenceCalculation = SentenceCalculation(sentence)
+    val numberOfDaysToSentenceExpiryDate = 0
+    sentenceCalculation.numberOfDaysToSentenceExpiryDate = numberOfDaysToSentenceExpiryDate
+    assertEquals(numberOfDaysToSentenceExpiryDate, sentenceCalculation.numberOfDaysToSentenceExpiryDate)
+  }
+
+  @Test
+  fun numberOfDaysToReleaseDate() {
+    val duration = Duration(2.0, ChronoUnit.DAYS)
+    val sentencedAt: LocalDate = LocalDate.of(2020, 1, 1)
+    val sentence = Sentence(duration, sentencedAt, 0, 0)
+    val sentenceCalculation = SentenceCalculation(sentence)
+    val numberOfDaysToReleaseDate = 0
+    sentenceCalculation.numberOfDaysToReleaseDate = numberOfDaysToReleaseDate
+    assertEquals(numberOfDaysToReleaseDate, sentenceCalculation.numberOfDaysToReleaseDate)
+  }
+
+  @Test
+  fun calculatedTotalRemandDays() {
+    val duration = Duration(2.0, ChronoUnit.DAYS)
+    val sentencedAt: LocalDate = LocalDate.of(2020, 1, 1)
+    val sentence = Sentence(duration, sentencedAt, 0, 0)
+    val sentenceCalculation = SentenceCalculation(sentence)
+    val calculatedTotalRemandDays = 0
+    sentenceCalculation.calculatedTotalRemandDays = calculatedTotalRemandDays
+    assertEquals(calculatedTotalRemandDays, sentenceCalculation.calculatedTotalRemandDays)
+  }
+
+  @Test
+  fun licenseExpiryDate() {
```

Comment:
minor one but license is spelled licence

Reply:
Nice spot! :)